lib/model/delete_button.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:note_app/database/sqldb.dart';
+
+import 'note_model.dart';
+
+class DeleteButton extends StatefulWidget {
```

Comment:
Why using `StatefulWidget` over `StatelessWidget `

Reply:
because we need a constructor
to define 'note' and 'onDelete voidCallBack'

Reply:
because we need a constructor
to define 'note' and 'onDelete voidCallBack'